Packs/PrismaCloud/TestPlaybooks/Prisma_Cloud_Test.yml

```diff
@@ -1,19 +1,20 @@
 id: Prisma Cloud V2 Test
-version: -1
+version: 15
```

Comment:
Please revert all changes that are not yours (you can compare to master in pycharm)